src/main/getDiForUnitTesting.ts

```diff
@@ -105,15 +130,16 @@ export const getDiForUnitTesting = (
     });
 
     di.override(loggerInjectable, () => ({
-      warn: noop,
-      debug: noop,
-      error: (message: string, ...args: any) => console.error(message, ...args),
-      info: noop,
+      warn: jest.fn(),
```

Comment:
Why are changing this to `jest.fn`? `noop` is enough and if you want to actually test it, then you should override it in the scope of that test.

Reply:
okay fair enough, I felt that just injecting the overridden version was fine. But I can remove.